README.md

```diff
@@ -1 +1,24 @@
-# talentuum-web
\ No newline at end of file
+# Talentuum Web 
+
+Basic web site based on https://github.com/Talentuum/website-static/ 
+
+## 🚀 Quick start
+
+
```

Comment:
Add the dependences time like `brew install gatsby-cli` and `npm install or yarn`

Reply:
Fixed on https://github.com/continuum/talentuum-web/pull/1/commits/1e208119438a6515befa18da739112d7363e751b